Scrot.py

```diff
@@ -0,0 +1,78 @@
+"""Extension wich basically wraps the commandline utility scrot to make
+screenshots from albert. The extension supports taking screenshots of the whole
+screen, an specific area or the current active window.
+
+When the screenshot was made you will hear a sound wich indicates that the
+screenshot was taken successfully.
+
+Screenshots will be saved in your home directory by default."""
+
+from albertv0 import *
+from shutil import which
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "scrot"
+__version__ = "1.0"
+__trigger__ = "scrot "
+__author__ = "Benedict Dudel"
+__dependencies__ = ["scrot", "xclip"]
+
+
+if which("scrot") is None:
+    raise Exception("'scrot' is not in $PATH.")
+
+if which("xclip") is None:
+    raise Exception("'xclip' is not in $PATH.")
+
+iconPath = iconLookup("camera-photo")
+fileName = "%Y-%m-%d-%T-screenshot.png"
```

Comment:
I think the home dir (or directory from which albert is started) is not a good place to store images. I think /tmp would be a better place but the best option is maybe to use the XDG directory for pictures: https://wiki.archlinux.org/index.php/XDG_user_directories